src/sources/socket/udp.rs

```diff
@@ -84,9 +83,7 @@ pub fn udp(
                         emit!(SocketEventReceived { byte_size,mode:SocketMode::Udp });
 
                         tokio::select!{
-                            result = out.send(event).compat() => {
-                                out = result?;
-                            }
+                            _ = out.send(event) => { }
```

Comment:
Can this return `ClosedError`?

Reply:
Yes it can! I will change it to handle that.